salt/modules/saltcheck.py

```diff
@@ -275,9 +298,12 @@ def state_apply(state_name, **kwargs):
         salt '*' saltcheck.state_apply postfix
     '''
     # A new salt client is instantiated with the default configuration because the main module's
-    # client is hardcoded to local
-    # If the minion is running with a master, a non-local client is needed to lookup states
-    caller = salt.client.Caller()
+    #   client is hardcoded to local
+    # If the minion is running with a master, a potentially non-local client is needed to lookup states
+    local_opts = salt.config.minion_config(__opts__['conf_file'])
+    if '_salt/running_data/var/run/salt-minion.pid' in __opts__.get('pidfile', False):
```

Comment:
Not sure what the best way is of detecting a salt-ssh session. Advice gladly accepted